draft-irtf-cfrg-spake2.xml

```diff
@@ -199,17 +216,22 @@
             data AAD, is as follows.</t>
 
         <figure><artwork><![CDATA[
-        TT  -> Hash(TT) = Ka || Ke
-        AAD -> KDF(nil, Ka, "ConfirmationKeys" || AAD) = KcA || KcB
+    TT  -> Hash(TT) = Ka || Ke
+    AAD -> KDF(nil, Ka, "ConfirmationKeys" || AAD) = KcA || KcB
         ]]></artwork></figure>
 
         <t>A and B output Ke as the shared secret from the protocol. Ka and its derived keys are not
             used for anything except key confirmation.</t>
     </section>
-    <section title="Per-user M and N">
+    <section title="Per-User M and N">
       <t>
       To avoid concerns that an attacker needs to solve a single ECDH instance to break the authentication of SPAKE2, a variant
-      based on using <xref target="I-D.irtf-cfrg-hash-to-curve"/> is also presented. In this variant M=h2c(HASH(A, B, "M for SPAKE2"), N=h2c(HASH(A, B, "N for SPAKE2")). In addition M and N may be equal to have a symmetric variant. The security of these variants is examined in <xref target="MNVAR"/>
+      based on using <xref target="I-D.irtf-cfrg-hash-to-curve"/> is also presented. In this variant, M and N are computed as follows:
+        <figure><artwork><![CDATA[
+    M = h2c(Hash("M for SPAKE2" || len(A) || A || len(B) || B))
+    N = h2c(Hash("N for SPAKE2" || len(A) || A || len(B) || B))
```

Comment:
I see that we need to spell the function as Hash() and give only a single argument, but note that this changes the order in which the identities and context string are presented.  I don't think that's a problem, but note it as part of the review.